gateway-ha/pom.xml

```diff
@@ -137,6 +137,11 @@
             <artifactId>ehcache</artifactId>
             <version>${dep.ehcache.version}</version>
         </dependency>
+        <dependency>
```

Comment:
Had to add here since the transitive and implied availability no longer was valid .. this is temporary and will be removed again in the clean up of the gateway-ha module